test/inRange.js

```diff
@@ -0,0 +1,86 @@
+import chai from 'chai';
+
+import * as RA from '../src/index';
+import eq from './shared/eq';
+import MIN_SAFE_INTEGER from '../src/internal/polyfills/Number.MIN_SAFE_INTEGER';
+import MAX_SAFE_INTEGER from '../src/internal/polyfills/Number.MAX_SAFE_INTEGER';
+
+describe('inRange', function() {
+  it('tests whether an integer falls within supplied range', function() {
+    const f = RA.inRange(5, 10);
+    eq(f(5), true);
+    eq(f(9), true);
+    eq(f(10), false);
+    eq(f(4), false);
+    eq(f(11), false);
+  });
+
+  it('tests whether a float falls within supplied range', function() {
+    const f = RA.inRange(5.5, 10.5);
+    eq(f(5.5), true);
+    eq(f(9.5), true);
+    eq(f(10.5), false);
+    eq(f(4.5), false);
+    eq(f(11.5), false);
+  });
+
+  it('should support negative values', function() {
+    const f = RA.inRange(-10, -5);
+    eq(f(-6), true);
+    eq(f(-5), false);
+    eq(f(-10), true);
+    eq(f(-4), false);
+    eq(f(-11), false);
+  });
+
+  it('should support currying', function() {
+    eq(RA.inRange(5)(10)(5), true);
```

Comment:
I think to test the currying you have to test the arity combinations (as we do in other tests)
```js
eq(RA.inRange(5)(10)(5), true);
eq(RA.inRange(5, 10)(5), true);
eq(RA.inRange(5, 10, 5), true);
 eq(RA.inRange(5)(10, 5), true);
```
Do this pls and squash and merge. Will be pristine ;] Thanks for you patience